corehq/form_processor/models.py

```diff
@@ -51,7 +51,7 @@
 XFormOperationSQL_DB_TABLE = 'form_processor_xformoperationsql'
 
 CommCareCaseSQL_DB_TABLE = 'form_processor_commcarecasesql'
```

Comment:
why not also rename these other constants?

Reply:
Thanks, did this during the migration rework.